<launch>

  <param name="use_sim_time" value="false" />

  <node pkg="velodyne_height_map" type="heightmap_node" name="heightmap_node" output="screen">
    <remap from="/velodyne_points" to="merged_z_filtered_velodyne"/>
    <param name="height_threshold" value="0.5"/>
    <param name="cell_size" value="0.5"/>
    <!--param name="cell_size" value="0.5"/-->
    <param name="negative_threshold" value="-0.5"/>
    <!--param name="grid_dimensions" value="200"/-->
    <param name="grid_dimensions" value="600"/>

    <remap from="/map" to="/local_map" />

  </node>

</launch>
